docs/HERMES_AIR.md

```
@@ -0,0 +1,232 @@
+
+`sudo tailscale up` and complete the browser login. Confirm the Pro is also in the tailnet:
+
+```bash
+tailscale status | grep -E "(macbook-pro|hermes)"
+```
+
+### 3. Telegram bot creation
+
+In Telegram, chat with `@BotFather`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)